pkg/test/resource/resource.go

```diff
@@ -294,6 +297,14 @@ func MakeTCPListener(listenerName string, port uint32, clusterName string) *v2.L
 	}
 }
 
+// MakeRuntime creates an RTDS layer with no fields.
+func MakeRuntime(runtimeName string) *discovery.Runtime {
+	return &discovery.Runtime{
+		Name:  runtimeName,
+		Layer: &pstruct.Struct{},
```

Comment:
`MakeRuntime` currently just returns an RTDS layer with 0 fields defined.

Reply:
Addressed! Added two dummy fields :) Didn't felt it necessary to expose the fields on the function to keep the code simple, hope that's OK